rpcs3/Emu/Cell/Modules/cellOskDialog.cpp

```diff
@@ -51,10 +52,28 @@ void fmt_class_string<CellOskDialogContinuousMode>::format(std::string& out, u64
 	});
 }
 
+atomic_t<bool> g_osk_pointer_enabled = false;
```

Comment:
g_fxo instead of separate globals. Especially since this is not performance-critical code.

Reply:
moved it to osk_info